guacamole/src/main/webapp/app/login/styles/dialog.css

```diff
@@ -115,13 +115,6 @@
     background-image: url("images/guac-tricolor.png");
 }
 
-.login-ui.continuation div.login-dialog {
-    border-right: none;
-    border-left: none;
-    box-shadow: none;
-    max-width: 6in;
-}
-
```

Comment:
I could leave in the max-width option there and that would un-squish things a bit, if that's preferable.  I think the main thing was the shadow/box effect.